src/noit_metric_director.c

```diff
@@ -1232,6 +1239,8 @@ void noit_metric_director_init() {
     mtev_frrh_set_prob_function(search_miss_cache, generation_aware_displace);
   }
 
+  OpenSSL_add_all_digests();
```

Comment:
should be done only once

Reply:
I agree, I just didn't see one global place to do it. We have 3 main contexts: noitd, stratcond, and libnoit. I couldn't find a common place to initialize this that would work for all. Additionally `test/test_tags.c` is a test-only binary that will eventually call into the `noit_metric.c` code but I don't want to have to add libcrypto to it just for that.

Reply:
I think the most logical place to initialize digests would be in libnoit. Both noitd and stratcond build libnoit objects in, so all three contexts would be covered. However, there doesn't seem to be a good place to put this so I might need to add an object.

Reply:
I think the most logical place to initialize digests would be in libnoit. Both noitd and stratcond build libnoit objects in, so all three contexts would be covered. However, there doesn't seem to be a good place to put this so I might need to add an object.

Reply:
Removed in favor of a single call site.